website/content/en/status/report-2025-01-2025-03/clusteradm.adoc

```diff
@@ -0,0 +1,61 @@
+=== Cluster Administration Team
+
+Links: +
+link:https://www.freebsd.org/administration/#t-clusteradm[Cluster Administration Team members] URL: link:https://www.freebsd.org/administration/#t-clusteradm[]
+
+Contact: Cluster Administration Team <clusteradm@FreeBSD.org>
+
+FreeBSD Cluster Administration Team members are responsible for managing the machines the Project relies on to synchronize its distributed work and communications.
+
+In this quarter, the team has worked on the following:
+
+* Regular support for FreeBSD.org user accounts.
+* Regular disk and parts support (and replacement) for all physical hosts and mirrors.
+* Cluster software refresh.
+* Moving more cluster services to Chicago.
+* Supporting the Grimoirelab dashboard effort.
+* Coordinate community mirrors.
+
+==== Moving cluster services to Chicago
+
+We started building up our new site in Chicago 2024, with a long-term goal to have Chicago as our primary location.
+Since 2024Q4, we began decommissioning older machines in New Jersey and moving services to the newer machines in Chicago.
+In 2025Q1, we started upgrading critical services in the cluster and testing to setup in Chicago.
+
+==== Git web interface mirrors
+
+While the project's public read-only git repository is built by a globally distributed mirror, the web interface (cgit) is not.
+We found there is increasing requirement of accessing it, and for improving the response time and reilibility, we setup the cgit on the mirrors around the world.
+
```

Comment:
We also deployed [Anubis](https://anubis.techaro.lol/) (a lightweight proof-of-work scraper-bot mitigation tool) to protect the cgit interface from the excessive crawler traffic it began receiving in recent months.

Thanks @lwhsu  for preparing this.

Reply:
From the log I thought it belongs to Q2? Or it's started in Q1?

Reply:
ah, you are right, it was in April.